frb_example/pure_dart/rust/src/api.rs

```diff
@@ -959,6 +958,10 @@ pub fn nested_id(id: [TestId; 4]) -> [TestId; 2] {
     }
 }
 
+/// Opaque types
+pub trait DartDebug: DartSafe + Debug + Send + Sync {}
+impl<T: DartSafe + Debug + Send + Sync> DartDebug for T {}
+
```

Comment:
uber-nit: looks like this is a merge failure. The 3 lines were at 1006-1009 and there is no need to move them here in order to make diff clean :)